pilot/pkg/proxy/envoy/v2/config.go

```diff
@@ -41,19 +43,24 @@ import (
 )
 
 const (
-	// filterNameRouter is the name for the router filter.
-	filterNameRouter = "router"
+	// names for filters taken from envoy v2 API.
+	filterNameRouter            = "envoy.router"
+	filterNameCors              = "envoy.cors"
+	filterHTTPConnectionManager = "envoy.http_connection_manager"
```

Comment:
use go-control-plane variables? e.g. `cache.HTTPConnectionManager`

Reply:
It's ok to duplicate a bit - we don't want this to depend on go-control-plane just for a variable.

Reply:
We already depend on all protos in the library. Might as well add a string. This would save all the time debugging the wrong string you put in the original PR.